ofrak_core/ofrak/resource.py

```diff
@@ -747,15 +747,17 @@ def _view_as(self, viewable_tag: Type[RV]) -> Union[RV, Awaitable[RV]]:
             self._resource_view_context.add_view(self.get_id(), view)
             return cast(RV, view)
 
-        analysis_tasks = [
-            self._analyze_attributes(attrs_t)
-            for attrs_t, existing in zip(composed_attrs_types, existing_attributes)
-            if not existing
-        ]
+        analysis_task = self._analyze_attributes(
+            tuple(
+                attrs_t
+                for attrs_t, existing in zip(composed_attrs_types, existing_attributes)
+                if not existing
+            )
+        )
 
         # Only if analysis is absolutely necessary is an awaitable created and returned
         async def finish_view_creation() -> RV:
-            await asyncio.gather(*analysis_tasks)
+            await asyncio.gather(analysis_task)
```

Comment:
If there is just a single task, does it make sense to `asyncio.gather` it instead of just `await`ing it?

```suggestion
            await analysis_task
```